Incompatibility with ansible 2.5.1 ?

# Issue Type

- Bug report

# Molecule and Ansible details

```
ansible --version
```
2.5.1
```
molecule --version
```
2.12.1
Molecule installation method (one of):
- pip

Ansible installation method (one of):
- pip

# Desired Behavior

Testinfra tests should work fine.

# Actual Behaviour (Bug report only)

Launching Testinfra tests trigger the following error:

`    ============================= test session starts ==============================
    platform linux2 -- Python 2.7.12, pytest-3.5.0, py-1.5.3, pluggy-0.6.0
    rootdir: /home/stephen/dev/kafka-lib/molecule/kafka_1.0.1, inifile:
    plugins: testinfra-1.7.1
collected 0 items / 1 errors                                                   
    
    ==================================== ERRORS ====================================
    ____________________ ERROR collecting tests/test_default.py ____________________
    /usr/local/lib/python2.7/dist-packages/pluggy/__init__.py:617: in __call__
        return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
    /usr/local/lib/python2.7/dist-packages/pluggy/__init__.py:222: in _hookexec
        return self._inner_hookexec(hook, methods, kwargs)
    /usr/local/lib/python2.7/dist-packages/pluggy/__init__.py:216: in <lambda>
        firstresult=hook.spec_opts.get('firstresult'),
    /usr/local/lib/python2.7/dist-packages/_pytest/python.py:200: in pytest_pycollect_makeitem
        res = list(collector._genfunctions(name, obj))
    /usr/local/lib/python2.7/dist-packages/_pytest/python.py:378: in _genfunctions
        self.ihook.pytest_generate_tests(metafunc=metafunc)
    /usr/local/lib/python2.7/dist-packages/pluggy/__init__.py:617: in __call__
        return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
    /usr/local/lib/python2.7/dist-packages/pluggy/__init__.py:222: in _hookexec
        return self._inner_hookexec(hook, methods, kwargs)
    /usr/local/lib/python2.7/dist-packages/pluggy/__init__.py:216: in <lambda>
        firstresult=hook.spec_opts.get('firstresult'),
    /usr/local/lib/python2.7/dist-packages/testinfra/plugin.py:143: in pytest_generate_tests
        ansible_inventory=metafunc.config.option.ansible_inventory,
    /usr/local/lib/python2.7/dist-packages/testinfra/host.py:125: in get_hosts
        for backend in testinfra.backend.get_backends(hosts, **kwargs):
    /usr/local/lib/python2.7/dist-packages/testinfra/backend/__init__.py:89: in get_backends
        for name in klass.get_hosts(host, **kw):
    /usr/local/lib/python2.7/dist-packages/testinfra/backend/ansible.py:72: in get_hosts
        return AnsibleRunner(kwargs.get("ansible_inventory")).get_hosts(host)
    /usr/local/lib/python2.7/dist-packages/testinfra/utils/ansible_runner.py:171: in __init__
        self.cli._play_prereqs(self.cli.options))
    /usr/local/lib/python2.7/dist-packages/ansible/cli/__init__.py:804: in _play_prereqs
        inventory = InventoryManager(loader=loader, sources=options.inventory)
    /usr/local/lib/python2.7/dist-packages/ansible/inventory/manager.py:145: in __init__
        self.parse_sources(cache=True)
    /usr/local/lib/python2.7/dist-packages/ansible/inventory/manager.py:206: in parse_sources
        parse = self.parse_source(source, cache=cache)
    /usr/local/lib/python2.7/dist-packages/ansible/inventory/manager.py:261: in parse_source
        if plugin.verify_file(source):
    /usr/local/lib/python2.7/dist-packages/ansible/plugins/inventory/yaml.py:84: in verify_file
        if not ext or ext in self.get_option('yaml_extensions'):
    /usr/local/lib/python2.7/dist-packages/ansible/plugins/__init__.py:58: in get_option
        option_value = C.config.get_config_value(option, plugin_type=get_plugin_class(self), plugin_name=self._load_name, variables=hostvars)
    /usr/local/lib/python2.7/dist-packages/ansible/config/manager.py:284: in get_config_value
        value, _drop = self.get_config_value_and_origin(config, cfile=cfile, plugin_type=plugin_type, plugin_name=plugin_name, keys=keys, variables=variables)
    /usr/local/lib/python2.7/dist-packages/ansible/config/manager.py:304: in get_config_value_and_origin
        defs = self._plugins[plugin_type][plugin_name]
    E   KeyError: 'inventory'
    !!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
    =========================== 1 error in 0.89 seconds ============================
`
The exact same test with exact same configuration is OK with Ansible 2.5.0 (instead of 2.5.1).

--- a/.yamllint
+++ b/.yamllint
@@ -8,6 +8,4 @@ rules:
     max-spaces-inside: 1
     level: error
   line-length: disable
-  # NOTE(retr0h): Templates no longer fail this lint rule.
-  #               Uncomment if running old Molecule templates.
-  # truthy: disable
+  truthy: disable

--- a/.yamllint
+++ b/.yamllint
@@ -1,11 +1,16 @@
 extends: default
 
 rules:
-  braces:
-    max-spaces-inside: 1
-    level: error
-  brackets:
-    max-spaces-inside: 1
-    level: error
+
+  # Disable line-length and truthy values reporting
   line-length: disable
   truthy: disable
+
+  # Max 1 space to separate the elements in brakets
+  braces:
+    max-spaces-inside: 1
+
+  # Max 1 space in empty brackets
+  brackets:
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 1

--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -1,2 +1,2 @@
-molecule==2.11.0
+molecule==2.14.0
 docker-py==1.10.6

--- a/tox.ini
+++ b/tox.ini
@@ -18,8 +18,6 @@ deps =
     ansible22: ansible<2.3
     ansible23: ansible<2.4
     ansible24: ansible<2.5
-    # ansible25: ansible<2.6
-    # see https://github.com/metacloud/molecule/issues/1262
-    ansible25: ansible==2.5.0
+    ansible25: ansible<2.6
 commands =
     {posargs:molecule test --all --destroy always}

